Resources/Locale/en-US/_NF/cargo/price-gun-component.ftl

```diff
@@ -16,3 +16,6 @@ pirate-contraband-price-gun-pricing-result = Skizzit's says {THE($object)} {$pri
 pirate-contraband-price-gun-verb-text = Appraisal
 pirate-contraband-price-gun-verb-message = Appraise {THE($object)}.
 pirate-contraband-price-gun-pricing-result-none = Skizzit's doesn't have an entry for {THE($object)}.
+
+# Appraisal cartridge
+appraisal-program-name = Appraisal App Plus
```

Comment:
Move to cartridges.ftl under _NF (Make a new file)

Reply:
Ah, thanks. Missed that bit, will fix it.

Reply:
Alright, moved that to an appropriate new file. Also made the PDA UI column labels localized instead of hard coded.